server/index.js

```diff
@@ -1,15 +1,15 @@
 const express = require('express');
 const createError = require('http-errors');
-const socketio = require('socket.io');
 const http = require('http');
 require('dotenv/config');
```

Comment:
Isn't [this](https://github.com/motdotla/dotenv#usage) the recommended usage? 

`require('dotenv').config()`

Reply:
Not done in the scope of this card.